src/ovp_pipeline/runtime_processes.py

```
@@ -0,0 +1,156 @@
+    except ValueError:
+        return command
+    if not parts:
+        return ""
+    try:
+        ovp_index = next(index for index, part in enumerate(parts) if Path(part).name == "ovp")
+    except StopIteration:
+        return " ".join(parts[1:]) if len(parts) > 1 else ""
+    args = parts[ovp_index + 1 :]
+    filtered: list[str] = []
+    skip_next = False
+    for index, part in enumerate(args):
```

Ruff (0.15.10)

[warning] Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)